vscripts/client/cl_fullscreen_map.nut

```diff
@@ -54,8 +56,22 @@ function FullscreenMinimapButtonReleased( player )
 	ResetAndShowMinimap( player )
 }
 
+function ToggleFullscreenMinimap( player )
+{
+	if ( !IsAlive( player ) || IsWatchingKillReplay() || !level.fullscreenMinimapEnabled || Riff_MinimapState() == eMinimapState.Hidden || 	PlayerHasPassive( player, PAS_MINIMAP_ALL ) )
+		return
+
+	if ( level.showingFullscreenMap )
+		ResetAndShowMinimap( player )
+	else
+		ShowFullMap( player )
+}
+
 function ShowFullMap( player )
 {
+	if ( level.showingFullscreenMap )
+		return
```

Comment:
Then for consistency with the other func (see the other comment) I'd say we shouldn't put this here either. Looking at these funcs, they should be idempotent, without any side effects from calling them multiple times